INSTALL.rst

```diff
@@ -138,8 +122,23 @@ PIConGPU Source Code
 Optional Libraries
 ^^^^^^^^^^^^^^^^^^
 
-If you do not install the optional libraries, you will not have the full amount of PIConGPU plugins.
-We recommend to install at least **pngwriter** and either **libSplash** (HDF5) or **ADIOS**.
+CUDA
+""""
+- `7.5+ <https://developer.nvidia.com/cuda-downloads>`_
+- required if you want to run on Nvidia GPUs
+- *Debian/Ubuntu:* ``sudo apt-get install nvidia-cuda-toolkit``
+- *Arch Linux:* ``sudo pacman --sync cuda``
+- *Spack:* ``spack install cuda``
+- at least one **CUDA** capable **GPU**
+- *compute capability*: ``sm_20`` or higher (for CUDA 9+: ``sm_30`` or higher)
+- `full list <https://developer.nvidia.com/cuda-gpus>`_ of CUDA GPUs and their *compute capability*
+- `More <http://www.olcf.ornl.gov/titan/>`_ is always `better <http://www.cscs.ch/computers/piz_daint/index.html>`_. Especially, if we are talking GPUs :-)
+- *environment:*
+
+  - ``export CUDA_ROOT=<CUDA_INSTALL>``
+
+If you do not install the following libraries, you will not have the full amount of PIConGPU plugins.
+We recommend to install at least **pngwriter** and either **libSplash** (+ **HDF5**) or **ADIOS**.
```

Comment:
no, this text only applies to the plugins as it mentions. CUDA is an optional backend for PIConGPU core now.